java/pending_code_changes.txt

```diff
@@ -1,11 +1,9 @@
-Pending metadata changes:
-* Metadata fixes for BY, KR, NA
-
-Pending code changes:
-* PhoneMetadata.java has two more fields to represent possible lengths of phone
+Code changes:
+- PhoneMetadata.java has two more fields to represent possible lengths of phone
```

Comment:
It's Phonemetadata.java with a lowercase 'm'.
Also, please leave a space before the '-' to be consistent with release notes:

```nomarkdown
Code changes:
 - Phonemetadata.java...
```

Also below.